### ISSUE TITLE
[iOS] Calling TrackPlayer.getCurrentTrack() resolves with null when no track is playing

### ISSUE THREAD

**Issue by anderslemke** [Guichaguri/react-native-track-player#36]
To reproduce:
```
await TrackPlayer.setupPlayer({});
TrackPlayer.getCurrentTrack();
```

I get this yellow box:

![screen shot 2017-09-04 at 11 33 12](https://user-images.githubusercontent.com/708095/30020671-f505c20e-9164-11e7-9509-cc72398fa7a0.png)

**Comment by dcvz** [comment]
That's not a crash.. `getCurrentTrack` is a promise. It resolves correctly if the player is playing or rejects if it's not (as there's no track). That error is telling you that you're not handling the possible promise rejection.

**Comment by anderslemke** [comment]
Yes. OK.

But then the API is not consistent between platforms. iOS just resolves with null, instead of rejecting.

**Comment by anderslemke** [comment]
How about `getPosition`, should that also reject if not playing?

**Comment by dcvz** [comment]
Fixed here: ef3cfc38216696b7d461c1668b4eee25d1bbe7ce

getPosition doesn't have it but maybe it should; I'll create a separate issue for that.
